waggle-dance-core/src/test/java/com/hotels/bdp/waggledance/mapping/model/DatabaseMappingImplTest.java

```diff
@@ -817,4 +818,15 @@ public void transformOutboundGetTableResultWithView() throws Exception {
     assertThat(transformedResult.getTable().getViewOriginalText(), is(VIEW_ORIGINAL_TEXT_TRANSFORMED));
   }
 
+  @Test
+  public void transformInboundPartitionValuesRequest() {
+    PartitionValuesRequest request = new PartitionValuesRequest();
+    request.setDbName(DB_NAME);
+    request.setTblName(TABLE_NAME);
```

Comment:
We normally also test something specific to this kind of request (i.e. more than just the table and db names). For this one I think we could use `setPartitionKeys` and then assert the values are the same after the transform. Some of the other tests above demonstrate this.